docs/src/pages/components/data-grid/sorting/sorting.md

```diff
@@ -57,34 +116,14 @@ const columns: GridColDef = [
 ];
 ```
 
-{{"demo": "pages/components/data-grid/sorting/OrderSortingPerColumnGrid.js", "bg": "inline"}}
-
-## Disable sorting
-
-By default, all columns are sortable.
-This can be revoked using the sortable property of the `GridColDef` interface:
-
-```tsx
-const columns: GridColDef = [{ field: 'name', sortable: false }];
-```
-
-{{"demo": "pages/components/data-grid/sorting/DisableSortingGrid.js", "bg": "inline"}}
+{{"demo": "pages/components/data-grid/sorting/OrderSortingPerColumnGrid.js", "bg": "inline", "defaultCodeOpen": false}}
```

Comment:
The example on top of it should be updated "quantity" column is not visible anymore

Reply:
Same remark in the code of the demonstration